src/web/src/components/Posts/GitHubMobile.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  createStyles,
+  makeStyles,
+  Theme,
+  ListSubheader,
+  Accordion,
+  Typography,
+  AccordionSummary,
+  AccordionDetails,
+} from '@material-ui/core';
+import Repos from './Repos';
+import Issues from './Issues';
+import PullRequests from './PullRequests';
+import Commits from './Commits';
+import Users from './Users';
+import { VscGithub } from 'react-icons/vsc';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '0',
+      display: 'flex',
+      borderLeft: '1.5px solid #999999',
+      flexDirection: 'column',
+      [theme.breakpoints.up('lg')]: {
+        width: '21rem',
+      },
+      color: theme.palette.text.secondary,
+    },
+    GitHubInfoContainer: {
+      margin: '2rem 0 0 1rem',
+    },
+    icon: {
+      fontSize: '4rem',
+      marginRight: '1rem',
+      verticalAlign: 'text-bottom',
+      paddingLeft: '6rem',
+      alignItems: 'right',
```

Comment:
Did you mean `textAlign`? This property is invalid.

Reply:
Fixed! Thanks :)